jenkins-scripts/tools/header_migration_script.sh

```diff
@@ -0,0 +1,500 @@
+#!/usr/bin/env bash
+# Copyright (C) 2022 Open Source Robotics Foundation
```

Comment:
Maybe adding a `set -e` to stop on errors at the begging can help to detect hidden problems.

Reply:
Good call! https://github.com/ignition-tooling/release-tools/pull/712/commits/cc67a0b0449189078bfeecf0199a29b328461477